### ISSUE TITLE
Issues with borders

### ISSUE THREAD

**Issue by pavle-lekic-htec** [diegomura/react-pdf#302]
Hi,
I noticed some strange issues with borders. Sometimes they are not visible at all, sometimes are of different width even if I set the same width to all borders, sometimes they stretch after the end of the View (See image below)
![screen shot 2018-09-10 at 3 22 48 pm](https://user-images.githubusercontent.com/32193333/45300175-f0906f80-b50d-11e8-9d6c-b46e00402043.png)

This is the table component that I made to render tables:

```
import React from 'react';
import PropTypes from 'prop-types';
import { View, StyleSheet, Text } from '@react-pdf/renderer';

const tableBorderWidth = '1px';
const tableBorderColor = 'grey';

const styles = StyleSheet.create({
    rowsContainer: {
        flex: 0,
        flexDirection: 'column',
        borderTop: `${tableBorderWidth} solid ${tableBorderColor}`
    },
    row: {
        flex: 0,
        flexDirection: 'row',
        borderLeft: `${tableBorderWidth} solid ${tableBorderColor}`,
        borderBottom: `${tableBorderWidth} solid ${tableBorderColor}`
    },
    defaultText: {
        padding: 0,
        fontSize: 7
        // marginTop: 2 // hack to center text vertically,
    },
    defaultCell: {
        paddingVertical: 1,
        paddingHorizontal: 3,
        borderRight: `${tableBorderWidth} solid ${tableBorderColor}`
    }
});

export default class PDFTable extends React.PureComponent {
    renderSingleCell = ({ cellStyle, textStyle, value }, columnIndex) => (
        <View
            key={columnIndex}
            style={[styles.defaultCell, cellStyle, { width: this.props.columnWidths[columnIndex] }]}
        >
            <Text style={[styles.defaultText, textStyle]}>{value}</Text>
        </View>
    );

    renderSingleRow = (row, rowIndex) => {
        return (
            <View key={rowIndex} style={styles.row}>
                {row.map(this.renderSingleCell)}
            </View>
        );
    }

    render = () => {
        return (
            <View style={styles.rowsContainer}>
                {this.props.rows.map(this.renderSingleRow)}
            </View>
        );
    }
}

const CellShape = PropTypes.shape({
    cellStyle: PropTypes.objectOf(PropTypes.any),
    textStyle: PropTypes.objectOf(PropTypes.any),
    value: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
});

PDFTable.propTypes = {
    columnWidths: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
    rows: PropTypes.arrayOf(PropTypes.arrayOf(CellShape)).isRequired
};
```

Btw, I was only able to set border in pixels, is there any other unit I can use to define border?
Thanks.

**Comment by diegomura** [comment]
@pavle-lekic-htec can you check if this still happens in last version (1.0.0-alpha.17)?

**Comment by tankers746** [comment]
@diegomura can confirm this is still an issue on 1.5.4

**Comment by diegomura** [comment]
It’s not. Please provide a snippet if you think it is

**Comment by klis87** [comment]
@diegomura I also confirm it. see table example here: https://codesandbox.io/s/6xzq3rw8rk

**Comment by jani789** [comment]
increase table height

**Comment by Shivamaggw** [comment]
This issue is still persisting. However, I've found some even drilled-down points to regenerate the issue

**Version:** 
"@react-pdf/renderer": "^3.1.14",

**Findings**
- The PDF works fine on Chrome, Mac PDF engine
- The issue OCCURS in Adobe Acrobat PDF reader on Windows. Checked on Adobe version 2024.002.20857 | 64 bit. However, I believe this happens in all recent previous versions as well.

@diegomura @jani789 @klis87 @tankers746 - Did you find any fixes to your issues?
